### PR TITLE
meshery installation with helm is added

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

#### Description

This PR fixes #    

#### Checklist

- [] [DCO](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) signed.
- [] Title of the PR starts with area name, e.g. `meshery:`, `mesheryctl:`, `ui:`.

### CODE REVIEW

File: meshery [hunk 1]
```diff
@@ -62,16 +62,15 @@ unzip ${PWD}/meshery.zip
 
 printf "\nInstalling mesheryctl in /usr/local/bin.\n"
 WHOAMI=$(whoami)
-if [ $OSEXT = "Linux" ] ; then
-  MV_CMD=$(sudo mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl)
+if mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply meshery]: Good question, @Aisuko. This change really shouldn't be in this PR as it was done by @nupur-thakur elsewhere. @gunjanmimo will you update your fork with latest from upstream?

File: docs/pages/installation/kubernetes.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ permalink: installation/kubernetes
 ---
 
 # Quick Start with Kubernetes
+## <strong>Using Helm</strong>
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply docs/pages/installation/kubernetes.md]: @Aisuko, yes, that make sense to me.

File: meshery (hunk 1 above)
[the remark this answers is not in the document]
Comment (gunjanmimo) [reply meshery]: sir, is
 if [ $OSEXT = "Linux" ] ; then
  MV_CMD=$(sudo mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl)

this the latest upstream?

[the remark this answers is not in the document]
Comment (gunjanmimo) [reply meshery]: @leecalcote sir, i have updated

File: meshery [hunk 3]
```diff
@@ -64,14 +64,14 @@ printf "\nInstalling mesheryctl in /usr/local/bin.\n"
...
+  echo "mesheryctl installed."
 else
-  MV_CMD=$(mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl)
-fi
-if $MV_CMD ; then
-  echo "mesheryctl installed"
-else
-	echo "Cannot install mesheryctl. Check permissions of $WHOAMI for /usr/local/bin."
-  exit 1
+  if sudo mv ${PWD}/mesheryctl /usr/local/bin/mesheryctl ; then
```
Comment (Aisuko) [inline meshery]: If there is a new feature, please open a new PR with this content.

File: docs/pages/installation/kubernetes.md [hunk 4]
```diff
@@ -5,6 +5,15 @@ permalink: installation/kubernetes
...
+<strong>Helm v3 - run the following:</strong>
```
Comment (Aisuko) [inline docs/pages/installation/kubernetes.md]: The style should follow  `## <strong>Using Kubernetes cluster</strong>` right?

File: docs/pages/installation/kubernetes.md [hunk 5]
```diff
@@ -5,6 +5,15 @@ permalink: installation/kubernetes
...
+
+```
+  $ git clone https://github.com/layer5io/meshery.git; cd meshery
+  $ kubectl create namespace meshery
+  $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
+```
+## <strong>Using Kubernetes cluster</strong>
```
Comment (Aisuko) [inline docs/pages/installation/kubernetes.md]: I have a concern about this line, Should we change it to `## <strong>Deploying by YAML file</strong>`. I mean either Helm or YAML all deploy meshery on Kubernetes. So, `Using Kubernetes cluster ` may not make sense.

File: meshery (hunk 3 above)
Comment (leecalcote) [reply meshery]: @gunjanmimo will you remove this change from this PR? I think this was just an accidental inclusion, right?

File: docs/pages/installation/kubernetes.md [hunk 6]
```diff
@@ -5,6 +5,15 @@ permalink: installation/kubernetes
 ---
 
 # Quick Start with Kubernetes
+## <strong>Using Kubernetes cluster</strong>
+<strong>Helm v3 - run the following:</strong>
```
Comment (kushthedude) [inline docs/pages/installation/kubernetes.md]: Use markdown syntax here remove the `<strong` tags.

**Pull request merged**